python/core/auto_generated/mesh/qgsmeshtimesettings.sip.in

```diff
@@ -61,6 +61,20 @@ Returns number of offset hours for relative time formatting
     void setRelativeTimeOffsetHours( double relativeTimeOffsetHours );
 %Docstring
 Sets number of offset hours for relative time formatting
+%End
+
+    double datasetPlaybackInterval() const;
```

Comment:
Please remove from the bindings before merge!

Reply:
oh, thats right.... @MUrbanLutraConsulting you need to use `SIP_SKIP` and rerun sipify script.

Reply:
Modified